### PR TITLE
Test error handling

### PR DESCRIPTION
**Pull request by luisfelipesdn12**
I two tests for the `Expected an array of numbers` and `Expected a number` errors.

### CODE REVIEW

File: test.js [hunk 1]
```diff
@@ -10,3 +10,39 @@ test("main", t => {
...
+test("Expected an array of numbers error", t => {
+	// Each test case is an array to be unpacked
+	// and used as a list of arguments
+	const testCases = [
+		[42],
+		[-15, 1, 2, 4, 3, 5, 8, 6, 7, 9, 10, 25],
+		[{ values: [-15, 1, 2, 4, 3, 5, 8, 6, 7, 9, 10, 25] }]
+	]
+
+	t.plan(testCases.length * 2)
```
Comment (Richienb) [inline test.js]: I don't think we need `t.plan`.

**Review by Richienb — CHANGES_REQUESTED** [review]

File: test.js (hunk 1 above)
Comment (Richienb) [reply test.js]: https://github.com/avajs/ava/blob/master/docs/recipes/when-to-use-plan.md

Comment (luisfelipesdn12) [reply test.js]: Thank you for the warning! That stretch is clear about that:

> Tests with assertions inside callbacks, `if`/`then` statements, `for`/`while` loops, and (in some cases) `try`/`catch` blocks, are all good candidates for `t.plan()`.

I'll fix it.

File: test.js [hunk 2]
```diff
@@ -10,3 +10,39 @@ test("main", t => {
...
+
+	for (const testCase of testCases) {
+		const error = t.throws(() => boxPlot(...testCase))
```
Comment (Richienb) [inline test.js]: We can actually just pass the expected error message to `t.throws` directly. For example:

```js
t.throws(() => {...}, {
	message: "Error message"
})
```

Comment (luisfelipesdn12) [reply test.js]: Ok

**Comment by luisfelipesdn12** [comment]
All requested changes were made!

**Pull request merged**